aries-site/src/pages/templates/filtering.mdx

```diff
@@ -1,124 +1,122 @@
+import { Text } from 'grommet';
 import { Example } from '../../layouts';
 import {
   FilteringCards,
   FilteringLists,
   FilteringTable,
-  FilteringWithRangeSelector,
-  FilteringWithSelect,
+  QuickFilterToolbar,
 } from '../../examples';
 
+export const DataSummaryExample = ({ filters }) => (
+  <Text>
+    {filters ? 'X results of ' : undefined}Y items
+    <Text margin={{ horizontal: 'xsmall' }}>|</Text>Z selected
+  </Text>
+);
+
+For details on the components that make up the filtering experience, see [Data documentation](/components/data).
+
 <Example
   code={[
     'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/filtering/FilteringCards/FilteringCards.js',
     'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/filtering/FilteringCards/mockData.js',
   ]}
+  docs="https://v2.grommet.io/data"
   figma="https://www.figma.com/file/EA0OZFdocKVcQ55c7Jainb/HPE-Filter-Templates?node-id=681%3A1430"
-  showResponsiveControls={['laptop', 'mobile']}
+  showResponsiveControls={['fullScreen']}
   template
 >
   <FilteringCards />
 </Example>
 
-## What makes up filtering?
+## When to use
 
-Filtering allows data to be narrowed down around a set of specified parameters. When users may not have a specific record in mind but are seeking to group data around a certain set of attributes, filtering should be used to allow the user to narrow the data and take action on relevant records.
+Filtering allows data to be narrowed down around a set of specified parameters. When users may not have a specific record in mind but are seeking to group data around a certain set of attributes, filtering allows the user to narrow the data and take action on relevant records.
 
 If a user will likely be looking for a specific record as opposed to groups of data, consider implementing search.
 
-### Controls
+## Placement
 
 Filter controls should live above the data content and facilitate a user's ability to locate and take actions against records of interest.
 
-### Filters
+## Presentation
+
+The way filters are displayed is dependent on the number of filters, data types of the content, and the use case.
 
-The way filters are displayed is dependent on the use case, number of filters, and data types of the content. These [general guidelines](#guidelines) are a starting point for how to present filtering in your application, but it's important to consider each recommendation in the context of your application's use cases.
+- In most cases when there are many available filters, they should be [presented in a layer](#filters-in-a-layer).
```

Comment:
should we be specific as to which type of layer?

Reply:
Yes, I'll say "side drawer layer"

Reply:
Done.